test/src/main/resources/org/apache/accumulo/test/v2_import_test/data/distcp.txt

```diff
@@ -0,0 +1,24 @@
+====
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+====
```

Comment:
This header will break. The exception in the pom.xml needs to be updated instead of adding a license header to this data file.

Other exceptions/exclusions statements in the pom.xml files should be checked and updated, as needed, for any other moved files as well.